Shortest Distance Between Words #1685

ErzaTitania-2001 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

(Write your answer here.)

### Checklist

- [ y] I've read the contribution guidelines.
- [ y] I've checked the issue list before deciding what to submit.
- [ y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Shortest Distance Between Words #1685 as assigned by @shraddhavp

ErzaTitania-2001 committed:
Shortest Distance Between Words 

Shortest Distance Between Words #1685

ErzaTitania-2001 committed:
Update and rename GC2.java to Shortest_Dist.java

ErzaTitania-2001 committed:
Update README.md

ErzaTitania-2001 commented:
@iamrajiv.

vatsalkesarwani12 commented on Java/cp/Shortest_Dist.java:
```diff
@@ -0,0 +1,86 @@
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the list of words " + "\n" + "Words = ");
+        arr = new ArrayList < > ();
```
remove spaces between <>

ErzaTitania-2001 replied:
@plazzy99 done

ErzaTitania-2001 committed:
Removed spaces between < >

vatsalkesarwani12 reviewed (approved):
👍🏻

atarax665 commented on Java/cp/Shortest_Dist.java:
```diff
@@ -0,0 +1,86 @@
+
+        List < String > wd = new ArrayList(arr);
+        while (t > 0) {
+            String wd1 = sc.next();
```
Prompt user to enter the words between which the distance is to be determined.

ErzaTitania-2001 replied:
@atarax665 done.

atarax665 reviewed (changes requested):
Add comments wherever needed, it will help people to understand the approach used.

ErzaTitania-2001 committed:
Update Shortest_Dist.java

Added Comment lines
Prompted user to enter the words between which the distance is to be determined

ErzaTitania-2001 commented:
@iamrajiv  what is DeepSource error?

iamrajiv committed:
formatted code

iamrajiv commented:
@ErzaTitania-2001 DeepSource is a code analysis tool. Do not worry about that. It is yet to be set for repository properly.
